spec/spec.yaml

```
@@ -3071,3 +3071,38 @@ paths:
           description: error
           schema:
             $ref: "./definitions.yaml#/definitions/V4GenericResponse"
+
+  /v5/exception-notifications/:
+    post:
+      operationId: createExceptionNotification
+      x-internal: true
+      tags:
+        - exception notification
```

This tag has to be defined in the `tags` section. And it should be plural.
```suggestion
        - exception notifications
```